task/docker/0.1/README.md

```diff
@@ -0,0 +1,49 @@
+# Docker
+
+## Docker-Build
+
+The Docker Build task builds source into a container image using [Docker](https://github.com/docker/). This command executes the directives in the `Dockerfile` to assemble a container image, then pushes that image to a container registry.
+
+### Install the Docker Build Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/docker-build/docker-build.yaml
```

Comment:
Can you please update the install  path command as well like

kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/docker-build/0.1/docker-build.yaml